### PR TITLE
Hook up the endpoints and add pagination

### PR DESCRIPTION
**Pull request by RichieAHB**
Sorry this one is so big - bit of a rush to get it done!

This adds paginated tables, and hooks up the endpoints to the reducers. It's actually pretty simple despite the amount of code changes.

### CODE REVIEW

File: public/js/components/Table/PaginatedTable.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    get prevDisabled() {
+        return this.state.pageIndex < 1;
+    }
+
+    get nextDisabled() {
+        return this.state.pageIndex > this.pageCount - 2;
+    }
+
+    get visiblePageIndices() {
+        const { pagesAround } = this.props;
```
Comment (susiecoleman) [inline public/js/components/Table/PaginatedTable.js]: Can you remind me what the `{}` does in this context?

File: public/js/reducers/articleWordCountsReducer.js [hunk 2]
```diff
@@ -29,10 +8,7 @@ const initialState = {
         commissionedLength: number,
```
Comment (susiecoleman) [inline public/js/reducers/articleWordCountsReducer.js]: No comma after `number`

File: public/js/components/Table/PaginatedTable.js (hunk 1 above)
Comment (RichieAHB) [reply public/js/components/Table/PaginatedTable.js]: It's equivalent to `const pagesAround = this.props.pagesAround;`

Comment (susiecoleman) [reply public/js/components/Table/PaginatedTable.js]: 👍

File: public/js/components/ArticleWithoutCommissionedLengthTable.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        <PaginatedTable pageSize={10} pagesAround={0} alternate head={
+            <Tr>
+                <Th>Headline</Th>
+                <Th></Th>
+                <Th>Word Count</Th>
+            </Tr>
+        }>
+            {articles.map((x, i) => 
+                <Tr key={x.path || i}>
+                    <Td><a href={`https://www.theguardian.com/${x.path}`} target="_blank" rel="noopener noreferrer">{x.headline}</a></Td>
```
[the remark this answers is not in the document]
Comment (RichieAHB) [reply public/js/components/ArticleWithoutCommissionedLengthTable.js]: Yeah you're right ... I'll make a small GuPaths util

File: public/js/components/ArticleWithoutCommissionedLengthTable.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    <div>
+        <h3>Articles without a Commissioned Length</h3>
+        <PaginatedTable pageSize={10} pagesAround={0} alternate head={
+            <Tr>
+                <Th>Headline</Th>
+                <Th></Th>
+                <Th>Word Count</Th>
+            </Tr>
+        }>
+            {articles.map((x, i) =>
```
[the remark this answers is not in the document]
Comment (RichieAHB) [reply public/js/components/ArticleWithoutCommissionedLengthTable.js]: 👍

File: public/js/components/ArticleWithCommissionedLengthTable.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+        <PaginatedTable pageSize={10} pagesAround={0} alternate head={
+            <Tr>
+                <Th>Headline</Th>
+                <Th></Th>
+                <Th>Word Count</Th>
+                <Th>Commissioned Word Count</Th>
+                <Th>Number of words over</Th>
+            </Tr>
+        }>
+            {articles.map(x =>
```
[the remark this answers is not in the document]
Comment (RichieAHB) [reply public/js/components/ArticleWithCommissionedLengthTable.js]: 👍

**Pull request merged**